src/sentry/api/serializers/models/release.py

```diff
@@ -165,9 +165,9 @@ def get_attrs(self, item_list, user, *args, **kwargs):
         if project:
             project_ids = [project.id]
         else:
-            project_ids = ReleaseProject.objects.filter(release__in=item_list).values_list(
+            project_ids = list(ReleaseProject.objects.filter(release__in=item_list).values_list(
```

Comment:
Should use a `set` here to remove dupes. Just to make sure.

Reply:
Or I guess chaining on `.distinct()` into the QuerySet. I assume without this, you'll probably have a lot of rows.